src/app/services/wallet.service.ts

```diff
@@ -187,18 +187,14 @@ export class WalletService {
           inputs: txInputs,
           outputs: txOutputs,
           hoursSent: hoursToSend,
-          hoursBurned: totalHours - calculatedHours
+          hoursBurned: totalHours - calculatedHours,
+          row: this.generateRawTransaction(txInputs, txOutputs)
```

Comment:
The name must be changed here too, or it will not be possible to send transactions, because `encoded` will always be `undefined`.

Reply:
Sorry for the inattention, fixed.